components/ide/jetbrains/image/status/hot-swap.sh

```diff
@@ -1,45 +0,0 @@
-#!/bin/bash
```

Comment:
this one we should keep but fix later cc @andreafalzetti @felladrin

Reply:
You mean `components/ide/jetbrains/launcher/hot-deploy.sh` [[1](https://github.com/gitpod-io/gitpod/pull/15308/files/b69c1dff1608681330e8d2b6332ae7789d4442a4#diff-976f1dfa9e5c722d2b812509cebfe510cec3dde67e89ad3aa10e4045e674d77e)]? If so, we kept it.

Reply:
no, not-swap as well, it allows to iterate on jb-launcher within a running workspace